### PR TITLE
stm32: 0.22 doc updates for nucleo-l552ze, Swan, Lora-E5, nucleo-wl55jc

### PR DESCRIPTION
**Pull request by kenbell**
I've not included a general update for all the other boards - probably warrants someone doing a mass-update once all the new boards are merged.

Fixes part of: https://github.com/tinygo-org/tinygo-site/issues/226

### CODE REVIEW

**Comment by kenbell** [comment]
I've included a fix for the tool which doesn't seem compatible with the current TinyGo cache folder structure.

**Comment by sago35** [comment]
@kenbell 
I tried to change the base branch, but it did not work.
Can you please remove the following commit?
The following commits have been merged into the release branch.

https://github.com/tinygo-org/tinygo-site/pull/240/commits/5ba9a7402ae0ccb71ee3a8676ba5daf7165bee4e

File: content/docs/reference/microcontrollers/lorae5.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+[Documentation for the machine package for the LoRa E5](../machine/lorae5)
+
+## Flashing
+
+### OpenOCD
+
+Programs are loaded onto the LoRa-E5 with an STLink hardware programmer, using the `openocd` command line utility program to perform the board flashing. You must install [OpenOCD](http://openocd.org/) before you will be able to flash the LoRa-E5 board with your TinyGo code.
+
+- Connect an ST-Link device to the SWD/SWIN headers.
```
Comment (sago35) [inline content/docs/reference/microcontrollers/lorae5.md]: SWD/SWIM?
https://www.seeedstudio.com/LoRa-E5-Dev-Kit-p-4868.html

File: content/docs/reference/microcontrollers/swan.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Blues Wireless Swan"
+weight: 3
+---
+
+The [Swan](https://blues.io/products/swan/) is an ARM development board based on the ST Micro [stm32l4r5zi](https://www.st.com/resource/en/datasheet/stm32l4r5zi.pdf) SoC.
```
Comment (sago35) [inline content/docs/reference/microcontrollers/swan.md]: I think the following links are good to go along with the other pages.

https://www.st.com/en/microcontrollers-microprocessors/stm32l4r5zi.html

File: content/docs/reference/microcontrollers/lorae5.md (hunk 1 above)
Comment (kenbell) [reply content/docs/reference/microcontrollers/lorae5.md]: Thanks - have fixed.

File: content/docs/reference/microcontrollers/swan.md (hunk 2 above)
Comment (kenbell) [reply content/docs/reference/microcontrollers/swan.md]: Thanks - have changed the PDF link to be to the web page

**Comment by deadprogram** [comment]
Thanks very much for the updated @kenbell now merging.

**Pull request merged**